lstchain/scripts/lstchain_dvr_pixselector.py

```diff
@@ -362,17 +367,19 @@ def main():
                                                                   args.picture_threshold,
                                                                   camera_geom)
         else:
+            # For the creation of pixel masks, we get the most typical value
+            # of min_charge_for_certain_selection among the subruns of the run
+            # in the DVR_settings_LST*.h5 file
             runmask = dvr_settings['run_id'] == run_id
-            meanq = dvr_settings['min_charge_for_certain_selection'][runmask].mean()
```

Comment:
not needed the mean charge any more?

Reply:
this was only to obtain min_charge_for_certain_selection
Now it is done by get_typical_dvr_min_charge, so we do not need the meanq intermediate value
(not it is no longer the mean, but the mode, safer against carflashes etc)